sys/net/application_layer/dhcpv6/client.c

```diff
@@ -369,6 +369,15 @@ static bool _check_sid_opt(dhcpv6_opt_duid_t *sid)
             (memcmp(sid->duid, server.duid.u8, server.duid_len) == 0));
 }
 
+/* discard stale messages in the receive buffer */
+static void _flush_stale_replies(sock_udp_t *sock)
+{
+    int res;
+    while ((res = sock_udp_recv(sock, recv_buf, sizeof(recv_buf), 0, NULL)) > 0) {
```

Comment:
Probably not really going to happen, but better save than sorry
```suggestion
    while ((res = sock_udp_recv(sock, recv_buf, sizeof(recv_buf), 0, NULL)) >= 0) {
```